src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -0,0 +1,189 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import _ from 'lodash';
+import AutoComplete from './autoComplete';
+import Input from '../views/settings/components/forms/controls/input';
+
+class DropdownAutoCompleteMenu extends React.Component {
+  static propTypes = {
+    items: PropTypes.oneOfType([
+      // flat item array
+      PropTypes.arrayOf(
+        PropTypes.shape({
+          value: PropTypes.string,
+          label: PropTypes.node,
+        })
+      ),
+      // grouped item array
+      PropTypes.arrayOf(
+        PropTypes.shape({
+          value: PropTypes.string,
+          label: PropTypes.node,
+          items: PropTypes.arrayOf(
+            PropTypes.shape({
+              value: PropTypes.string,
+              label: PropTypes.node,
+            })
+          ),
+        })
+      ),
+    ]),
+    action: PropTypes.element,
+    isOpen: PropTypes.bool,
+    onSelect: PropTypes.func,
+  };
+
+  static defaultProps = {
+    onSelect: () => {},
+  };
+
+  filterItems = (items, inputValue) =>
+    items.filter(item => {
+      return (
+        [item.value, item.label]
+          .join(' ')
+          .toLowerCase()
+          .indexOf(inputValue.toLowerCase()) > -1
+      );
+    });
+
+  filterGroupedItems = (groups, inputValue) =>
+    groups
+      .map(group => {
+        return {
+          ...group,
+          items: this.filterItems(group.items, inputValue),
+        };
+      })
+      .filter(group => group.items.length > 0);
+
+  autoCompleteFilter = (items, inputValue) => {
+    let itemCount = 0;
+
+    if (items[0] && items[0].items) {
+      //if the first item has children, we assume it is a group
+      return _.flatMap(this.filterGroupedItems(items, inputValue), item => {
+        return [
+          {...item.group, groupLabel: true},
+          ...item.items.map(groupedItem => ({...groupedItem, index: itemCount++})),
+        ];
+      });
+    } else {
+      return this.filterItems(items, inputValue).map((item, index) => ({...item, index}));
+    }
+  };
+
+  render() {
+    let {onSelect, children, items, action, ...props} = this.props;
+
+    return (
+      <AutoComplete
+        itemToString={item => ''}
```

Comment:
This is so when we select an item, the search input becomes empty (instead of normal behavior where it autocompletes)